tests/test_hear_data_processing.py

```
@@ -0,0 +1,97 @@
+				"""--port""", str(_fixture_port_num),
+				"""--groups""", """'224.0.0.1'""",
+				"""--group""", """'224.0.0.1'"""
+			]
+			p = Process(
+				target=multicast.__main__.McastDispatch().doStep,
+				name="HEAR", args=("HEAR", _fixture_HEAR_args,)
+			)
+			p.start()
+			try:
+				while p.is_alive():
+					sender(group="224.0.0.1", port=_fixture_port_num, ttl=1, data="STOP Test")
```

Ruff

Undefined name `sender`

(F821)